hadoop-ozone/dist/src/main/smoketest/s3/commonawslib.robot

```diff
@@ -193,3 +193,22 @@ Revoke S3 secrets
     Execute and Ignore Error             ozone s3 revokesecret -y -u testuser
     Execute and Ignore Error             ozone s3 revokesecret -y -u testuser2
 
+Get bucket owner
+    [arguments]    ${bucket}
+    ${owner} =     Execute     ozone sh bucket info /s3v/${bucket} | jq -r '.owner'
+    [return]       ${owner}
```

Comment:
Please move this to `hadoop-ozone/dist/src/main/smoketest/ozone-lib/shell.robot`, since it's not S3-specific.